recipes/setup_raid_on_master.rb

```diff
@@ -57,9 +57,14 @@
   raid_dev = "/dev/md0"
 
   # Create RAID device with mdadm
+  raid_superblock_version = value_for_platform(
+    'centos' => { '>=8' => '1.2' },
```

Comment:
Is there a way to retrieve get the superblock version from OS/kernel dynamically? Do we expect this version to get updated in the future and might want to avoid hardcoding it?

Reply:
It's a good question that could be valid for all the OSes.

From the official [table](https://raid.wiki.kernel.org/index.php/RAID_superblock_formats#Sub-versions_of_the_version-1_superblock) it seems that these RAID formats are enough stable.
We might have new versions but I'd keep the work of retrieving the superblock from version as future work, if needed.